addons/point_of_sale/models/pos_order.py

```diff
@@ -46,9 +46,12 @@ def _order_fields(self, ui_order):
         }
 
     def _payment_fields(self, ui_paymentline):
+        payment_date = ui_paymentline['name']
+        if len(payment_date) > 10:
```

Comment:
Is there scenarios where you have a name with a size lower than ten? Do we get a date sometimes? This is the kind of if that needs to be documented.